CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ Main (unreleased)
   traces, metrics, and logs. Data can then be forwarded to other `otelcol`
   components. (@rfratto)
 
+- Expose Grafana Agent traces Kubernetes with port 4318 to OTLP exporter sustain both gRPC and HTTP protocols. (@Eji4h)
```

Comment:
While we appreciate the contribution, the CHANGELOG is reserved for user-facing code changes (meaning no docs, example manifests, etc.), so this wouldn't actually apply unfortunately.